src/main/java/org/sobotics/guttenberg/clients/Guttenberg.java

```diff
@@ -30,6 +30,23 @@
  * Fetches and analyzes the data from the API
  * */
 public class Guttenberg {	
+	
+	//Global Statistics
+	/**
+	 * The time when the last check was finished
+	 * */
+	public static Instant lastExecutionFinished = null;
+	
+	/**
+	 * The number of new answers that have been checked
+	 * */
+	public static int numberOfCheckedTargets = 0;
```

Comment:
Ideally, you'd want to use an `AtomicInteger` and, instead of a class with static fields, make that a property of some class that is injected in the other classes.